### ISSUE TITLE
Using a apostrophe in a tables makes the table crash

### ISSUE THREAD

**Issue by woutervuden** [floriankarsten/kirby-tablex#15]
When I use the  " ' " in the table the table crash when I save it. I end up with the tablex complete disappeared. I can't even add any columns and rows anymore. It's will return to it's empty state when switching panel pages back and forth.

**Comment by iskrisis** [comment]
Yes, i know about this. Unfortunately i haven't find any good easy fix at the time. It is combination of issues in vue.js templating and then saving output to yaml (where kirby saves data - txt files in content folder and quotes are used a lot).

So i think it is possible to change plugin to use single quotes or double quotes but not both at the same time. If you need something like that then try escaping with \' \" 

Also this plugin is outdated by now because i am putting time to kirby 3 fields so there won't be any fix from me (more like better table field for kirby 3 when it comes out).
